akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
Based on what I mentioned above (https://github.com/akka/akka/pull/23257#discussion_r125048369) and PR #23240, should I also rename ```use-role-set``` to ```use-roles``` ?